Update managing-channels.rst (#4787)
mattermost/docs: source/messaging/managing-channels.rst

Leaving a channel
-----------------

> Select the channel name at the top of the center pane to access the drop-down menu, then select **Leave Channel**. For Mattermost Desktop, you can also do that without entering the channel. If you wish to leave a channel, simply move your cursor on the top of the channel name at the sidebar. There, you can see the three dots icon (channel options) beside the channel name. Click the icon to access the drop down menu, then select **Leave Channel**. Any team member who leaves a Private channel must be re-added by another channel member if they wish to rejoin. Team members will not receive mention notifications from channels of which they are not members. 

Reviewer: ```suggestion
From within the channel, select the channel name at the top of the center pane to access the drop-down menu, then select **Leave Channel**. For Mattermost Desktop, you can hover over the channel name in the sidebar. Select **Channel Options** (the three dots), then select **Leave channel**. 

Any team member who leaves a Private channel must be re-added by another channel member if they wish to rejoin. Team members will not receive mention notifications from channels of which they are not members. 
```

Reviewer: This is a great addition. 

I've edited it for brevity, as we work on the principle of "less is more" in our docs. Hope that makes sense!

Use the **History** arrows at the top of the sidebar to move back and forth through your channel history. 

- Select the left arrow to go back one page. 

> - Select the right arrow to go forward one page. 

Reviewer: ```suggestion
- Select the right arrow to go forward one page.
```

Reviewer: We don't use trailing white spaces in our documentation - but I am happy to discuss the reason behind this change if you're open to it?

Author: It was an error from my side. Sorry for the whitespace.
